proposals/wac-ucr/scratch.md

```diff
@@ -0,0 +1,11 @@
+# Use Cases to Create
+
+Write use cases for:
+
+- Conditional Filter
+- Conditional Control
+- Conditional Action
+- Conditional Payment
+- UC Client Constraints
+
+DONE - Change VC requirement from FUTURE
```

Comment:
Added this file by mistake?

Reply:
yes lol 😑

Reply:
Cleared in https://github.com/solid/authorization-panel/pull/152/commits/01be17329981132f3454e3f0ff542c287113d687